lib/assets/maplibre/main.js

```diff
@@ -1,4 +1,5 @@
 import "https://unpkg.com/maplibre-gl@2.1.9/dist/maplibre-gl.js";
+import "https://loc8.us/maplibre-gl-vector-text-protocol/dist/maplibre-gl-vector-text-protocol.js";
```

Comment:
jsDelivr CDN [supports GitHub](https://www.jsdelivr.com/?docs=gh) :)

```suggestion
// TODO: point to npm when released
import "https://cdn.jsdelivr.net/gh/jimmyrocks/maplibre-gl-vector-text-protocol@73c5dc8f2bc3331298273e8800d6c5db4cdfffab/dist/maplibre-gl-vector-text-protocol.min.js";
```

Reply:
Thanks 🤗